src/helpers/utilities.js

```diff
@@ -25,3 +26,24 @@ export function ellipseText (text, maxLength = 9999) {
 export function ellipseAddress (address, width = 10) {
   return `${address.slice(0, width)}...${address.slice(-width)}`
 }
+
+export async function copyToClipboard (text) {
+  console.log('function')
```

Comment:
Why keep this `console.log` ?

Reply:
I was unsure about the Vue 2 call stack, needed a little refresher by trial and error. However those were deleted (see commit : 5f21ee14e6adcd61a5670b74329e636a5eac4d3b)